posthog/settings.py

```diff
@@ -25,11 +29,26 @@ def get_env(key):
     except KeyError:
         raise ImproperlyConfigured(f'The environment var "{key}" is absolutely required to run this software')
 
-# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+def get_list(text: str) -> List[str]:
```

Comment:
Methods like this could be refactored to a `settings.utils` module, because they are not part of the config. What do you think?

Reply:
I was inspired by the `get_env` function above this one. We could move those 3 methods to special utils file, however, I doubt if they will be reused anywhere outside the settings, what's your preference?

Reply:
> I doubt if they will be reused anywhere outside the settings, what's your preference?

Yep, which is why it makes sense to have a `settings` package here, and then have settings.utils here.

But this is also totally fine.